docs/locale/fr/LC_MESSAGES/cryptography.po

```diff
@@ -163,184 +160,183 @@ msgstr ""
+"keys (DEVICE_SIG_P_KEY, DEVICE_SIG_S_KEY) and use the authenticated "
 "channel to share their public parts with Alice."
 msgstr ""
 "Bob génère ses clés d'utilisateur (USER_ENC_P_KEY, USER_ENC_S_KEY) et de "
 "terminal (DEVICE_SIG_P_KEY, DEVICE_SIG_S_KEY) et utilise le canal "
 "authentifié pour communiquer leurs parties publiques à Alice."
 
-#: ../../cryptography.rst:39
+#: ../../cryptography.rst:43
 msgid ""
-"Alice signs that two keys with the help of her private key "
+"Alice signs those two keys with the help of her private key "
 "(DEVICE_SIG_S_KEY) and upload those certified keys to the metadata "
-"server. As each user key is signed by a device registered in the "
+"server. Since each user key is signed by a device registered in the "
 "organization and the one of the first user is signed by the root key "
 "(ORG_ROOT_SIG_S_KEY), re-validating the signature trust-chain, a client "
-"is able to verify that a key has been added to PARSEC by a legitimate "
-"terminal and can be considered as valid. Each user is attributed an email"
-" address at its creation in order to reflect his correspondence to a "
-"natural person. For a given email address, it exists at least one non-"
+"is able to verify that a key has been added to Parsec by a legitimate "
+"terminal and can be considered as valid. Each user is attributed an "
+"email address at its creation in order to reflect his correspondence to "
+"a natural person. For a given email address, it exists at least one non-"
 "revoked user in the organization (i.e. revocation of the existing user "
 "then creation of a new user with the same email address), while enabling "
 "other users to find him with the same email address."
 msgstr ""
-"Alice signe ces deux clés à l'aide de sa clé privée (DEVICE_SIG_S_KEY) et"
-" télécharge ces clés certifiées sur le serveur de métadonnées. Comme "
+"Alice signe ces deux clés à l'aide de sa clé privée (DEVICE_SIG_S_KEY) "
+"et télécharge ces clés certifiées sur le serveur de métadonnées. Comme "
 "chaque clé d’utilisateur est signée par un terminal enregistré dans "
 "l’organisation et celle du premier utilisateur est signée par la clé "
 "racine (ORG_ROOT_SIG_S_KEY), en revalidant la chaîne de signatures, un "
-"client est en mesure de s’assurer qu’une clé a bien été ajoutée à PARSEC "
+"client est en mesure de s’assurer qu’une clé a bien été ajoutée à Parsec "
 "par un terminal légitime et peut donc être considérée comme valide. Un "
-"utilisateur se voit attribuer une adresse email à sa création afin de "
-"signifier sa correspondance à une personne physique. Pour une adresse "
-"email donnée, il existe au plus un utilisateur non révoqué dans une "
+"utilisateur se voit attribuer une adresse e-mail à sa création afin de "
+"signifier sa correspondance à une personne physique. Pour une adresse e-"
+"mail donnée, il existe au plus un utilisateur non révoqué dans une "
 "organisation. De cette façon un utilisateur compromis peut être remplacé "
```

Comment:
> De cette façon un utilisateur compromis peut être remplacé

I can't find the equivalent part of that phrase in the original part above

Reply:
The equivalent part is missing in the original docs (`cryptography.rst`). When reading the translation, the French sentence seems more clear to me so I didn't spot the missing part. 

I think we should add it to `cryptography.rst` and then also to the `cryptography.po` so the French version is up-to-date :)

Do you want to propose these suggestions in gh ? Otherwise I'll add it locally